src/lib/support/Span.h

```diff
@@ -56,6 +60,20 @@ class Span
         return Span(mDataBuf + offset, length);
     }
 
+    // Allow converting a span with non-const T into a span with const T.
+    template <class U = T>
```

Comment:
The template shouldn't be needed here. Just use  `is_const<T>` in the `enable_if_t` below. Or it doesn't work?

Reply:
That doesn't work, for the reasons described in https://stackoverflow.com/a/52078691